services/ir/artifacts.mk

```diff
@@ -1,34 +1,13 @@
-# Get NeoFS IR artifacts (LOCODE database and NeoFS CLI)
+# Get NeoFS IR artifacts (NeOFS contracts, ADM and CLI)
 
-LOCODE_DB_ARCHIVE_PATH=./vendor
-LOCODE_DB_ARCHIVE_FILE=locode_db.gz
-
-get.ir: get.locode get.cli get.storage prepare.storage
-
-# Get NeoFS LOCODE database
-get.locode: LOCODE_DB_PATH?= 
-get.locode:
-	@mkdir -p ${LOCODE_DB_ARCHIVE_PATH}
-
-ifeq (${LOCODE_DB_PATH},)
-	@echo "⇒ Download NeoFS LOCODE database from ${LOCODE_DB_URL}"
-	@curl \
-		-sSL "${LOCODE_DB_URL}" \
-		-o ${LOCODE_DB_ARCHIVE_PATH}/${LOCODE_DB_ARCHIVE_FILE}
-else
-	@echo "⇒ Copy local archive of NeoFS LOCODE database from ${LOCODE_DB_PATH}"
-	@cp ${LOCODE_DB_PATH} ${LOCODE_DB_ARCHIVE_PATH}/${LOCODE_DB_ARCHIVE_FILE}
-endif
-
-	gzip -dfk ${LOCODE_DB_ARCHIVE_PATH}/${LOCODE_DB_ARCHIVE_FILE}
+get.ir: get.cli get.contracts get.adm get.storage prepare.storage
 
 # Download NeoFS CLI 
 .ONESHELL:
 get.cli: NEOFS_CLI_FILE=./vendor/neofs-cli
 get.cli: NEOFS_CLI_ARCHIVE_FILE=${NEOFS_CLI_FILE}.tar.gz
 get.cli: NEOFS_CLI_PATH?=
 get.cli:
-	@touch services/ir/.ir.env # https://github.com/docker/compose/issues/3560
```

Comment:
@vvarg229 i accidentally removed this line during rebase, seems like https://github.com/nspcc-dev/neofs-dev-env/pull/282#issuecomment-1861384504 caused by this